cmds/date/date.go

```diff
@@ -78,7 +90,7 @@ func formatParser(args string) []string {
 
 // replace map for the format patterns according POSIX and GNU implementations
 func dateMap(format string) string {
-	d := time.Now().In(z)
```

Comment:
i'd prefer not to add more global state. you could pass `t` to dateMap and the other thing?

Reply:
and now that we're here, the same could be said for `z` (i know it was this way before... but let's fix stuff like this instead of leaving it around)